yarn-project/circuit-types/src/interfaces/pxe.ts

```diff
@@ -73,7 +73,8 @@ export interface PXE {
    * the recipient's notes. We can send notes to this account because we can encrypt them with the recipient's
    * public key.
    */
-  registerRecipient(recipient: CompleteAddress): Promise<void>;
+  // TODO: #5834: FIX THIS AFTER THE COMPLETE ADDRESS REFACTOR
```

Comment:
```suggestion
  // TODO: #5834: Nuke publicKeys optional parameter after `CompleteAddress` refactor.
```
It was not clear to me initially what fix means.